server/modules/activityLog/activityLogger.js

```diff
@@ -48,6 +48,7 @@ const type = {
   userRemove: 'userRemove',
 
   // analysis
+  processingChainInsert: 'processingChainInsert',
```

Comment:
should we use processingChainCreate? it's more consistent with the other activity logs.. what do you think?

Reply:
ok, right